x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/action/filter/SecurityActionFilter.java

```diff
@@ -111,7 +111,7 @@ public SecurityActionFilter(AuthenticationService authcService, AuthorizationSer
                 listener.onFailure(e);
             }
         } else if (SECURITY_ACTION_MATCHER.test(action)) {
-            if (licenseState.isSecurityDisabledByLicenseDefaults()) {
+            if (licenseState.isSecurityEnabled()) {
```

Comment:
This one looks to be backwards.
```suggestion
            if (licenseState.isSecurityEnabled() == false) {
```